modules/network/multivpc/README.md

```diff
@@ -0,0 +1,136 @@
+## Description
+
+This module accomplishes the following:
+
+* Creates 2 to 8 [VPC networks][vpc]
+  * Each VPC contains exactly 1 subnetwork
+  * Each subnetwork contains distinct IP address ranges
+* Sets up the VPCs to use non-overlapping IP ranges
+* Outputs the `additional_networks` parameter, which is compatible with Slurm
+  modules
+
+There are 4 variables that differentiate this module from the standard VPC
+module.
+
+1. `network_prefix`: The name prefix of the VPCs to be created.  All
+   networks and subnetworks will start with this and end with a unique number.
+1. `network_count`: The number of VPCs to be created.
+1. `global_ip_address_range`: [CIDR-formatted IP range][cidr]
+1. `network_cidr_suffix`: The CIDR suffix that defines the address
+   space that the individual VPCs will cover.
+
+> [!WARNING]
+> The `network_cidr_suffix` should be always be larger than the CIDR suffix on
+> `global_ip_address_range`.  The difference between these two suffixes should
+> be large enough to accommodate the number of VPCs that are being deployed
+> (e.g. CIDR suffix bit difference <= `ceil(log2(network_count)))`).
+<!-- MD028/no-blanks-blockquote -->
+> [!NOTE]
+> For deployments that need multiple VPCs that do not meet this use-case, users
+> should deploy multiple individual VPC modules.
+
+[vpc]: ../vpc/README.md
+[cidr]: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation
+
+### Example
+
+This creates a new multi-VPC network named `multivpc-net`.
+
+```yaml
+  - id: multivpc1
+    source: modules/network/multivpc
+    settings:
+      network_prefix: multivpc-net
+      network_count: 8
+      global_ip_address_range: 172.16.0.0/12
+      network_cidr_suffix: 16
+
+  - id: network
+    source: modules/network/vpc
+
+(multivpc module renamed to multinetwork)
```

Comment:
I think conceptually the vpc module should precede the multivpc module, although it's unimportant technically.

Remove this line where I intended you to rename "multivpc1" to "multinetwork".

Reply:
Done